cellpose/gui/guiparts.py

```diff
@@ -408,10 +408,14 @@ class MinimapWindow(QDialog):
     def __init__(self, parent=None):
         super(MinimapWindow, self).__init__(parent)
         # Set the title of the window
-        self.title = "Minimap"
+        self.title = "Minimap (right click for options)"

```

Comment:
"for options" finde ich ein wenig schwammig, ich würde schon irgendwie erwähnen dass es um size options geht oder so damit das eindeutig ist.

Reply:
Verstehe ich, aber da das Menü erweiterbar ist mit weiteren "Optionen" habe ich da das verallgemeinert. Kann falls doch Bedarf ist aber auch leicht geändert werden.